app/controllers/replies_controller.rb

```diff
@@ -97,4 +97,8 @@ def find_new_thread
       @new_thread = NewThread.friendly.find_by_id(params[:new_thread_id])
     end
   end
+
+  def friendly_url
+    @new_thread = NewThread.friendly.find(params[:new_thread_id])
+  end
 end
```

Comment:
but `@new_thread` is supposed to find the threads related to the reply.i just replaced the statement that was previously used in `:create` and `:edit` actions in the before filter.